### ISSUE TITLE
event handler loss inside slides on breakpoints

### ISSUE THREAD

**Issue by advanced-media** [kenwheeler/slick#3604]
If you have direct bound event handlers inside slides (e.g. ajax-form at product-tiles in a product-carousel) and you use responsive configuration, the event handlers are wiped away.

===================================================================
#### Fiddle

https://jsfiddle.net/ntjovefz/

===================================================================
#### Steps to reproduce the problem

use slick v1.9.0
use responsive config
hit a breakpoint

===================================================================
#### What is the expected behaviour?

event handlers inside slides are NOT removed

===================================================================
#### What is observed behaviour?

event handlers inside slides are removed

===================================================================
#### More Details

Slick.prototype.cleanUpRows() 

Maybe it's a side effect of using .empty() as long as originalSlides are attached to the DOM:
> To avoid memory leaks, jQuery removes other constructs such as data and event handlers from the child elements before removing the elements themselves.
http://api.jquery.com/empty/

One can use .clone(true) or .detach() to build originalSlides collection.
https://stackoverflow.com/a/40369879

**Comment by DrezynSoft** [comment]
Try to use [events](https://github.com/kenwheeler/slick?tab=readme-ov-file#events). Event `init` and `breakpoint` are things you need probably.
